source/software/r_package_management.rst

```diff
@@ -105,17 +88,19 @@ If the result is blank, or reports that conda can not be found, modify
 the \`PATH\` environment variable appropriately by adding miniconda's bin
 directory to PATH.
 
-Creating a new conda environment is straightforward::
+The next step is to create a new conda environment which can be done as follows::
```

Comment:
some users are dummies. For such, we have to also mention that they need to pick a specific R version in advance, and substitute that below when `<version>`. Can you please include a sentence to cover this?